canonical-websites/snapcraft.io
# Removed notification bar on publisher pages

## QA
Pull the branch or visit the demo link

1. Open snapcraft.io
2. Login
3. Make sure there is no notification appearing in regards to updates on T&Cs on the published snaps summary page, listing pages or account details
